src/frontend/metrics/sendMetricsReport.ts

```diff
@@ -0,0 +1,53 @@
+import type {Jsonifiable, ReadonlyDeep} from 'type-fest';
+
+class SendMetricsReportHttpError extends Error {
+  readonly status: number;
+  readonly errorBody: string;
+  constructor({
+    status,
+    errorBody,
+  }: Readonly<{status: number; errorBody: string}>) {
+    super(`HTTP error sending metrics. Error message: ${errorBody}`);
+    this.status = status;
+    this.errorBody = errorBody;
+  }
+}
+
+/**
+ * Send a metrics report to the metrics server.
+ *
+ * Throws if the request fails due to a network error (e.g., no connection) or
+ * an HTTP error (e.g., a 403 status code).
+ */
+export default async function sendMetricsReport({
+  metricsUrl,
+  metricsApiKey,
+  metricsReport,
+  signal,
+}: ReadonlyDeep<{
+  metricsUrl: string;
+  metricsApiKey: string;
+  metricsReport: Jsonifiable;
+  signal: AbortSignal;
+}>): Promise<void> {
+  const body = new Blob([JSON.stringify({data: metricsReport})]);
```

Comment:
Is this working for you. I'm pretty sure you can't just use the browser Blob Api in RN and you will need to export an external library.

Reply:
It was working for me but I think it's cleaner without it anyway. Removed in 6a12ca145f2eb855fe289b2972047e0731aa0f7d.